data/tiddlers.json

```diff
@@ -329,9 +346,9 @@
         "faction_id": "1",
         "id": "1",
         "map_id": "1",
-        "modified": "20150706211307316",
-        "status": "à placer",
-        "tags": "zones",
+        "modified": "20150720105629062",
+        "status": "done",
+        "tags": "zones [[_placement à valider]]",
```

Comment:
Can you remove this tag and make it a `status` like `to_validate`? Else I'll have to update the script to explode the tags and find the "good ones".